vars/platform_vars.yml

```diff
@@ -0,0 +1,356 @@
+---
```

Comment:
As these are defaults in the Ansible Role anyway, I'm OK having these in a common path